donate/payments/views.py

```diff
@@ -785,6 +787,8 @@ def form_valid(self, form, send_data_to_basket=True):
         data = form.cleaned_data.copy()
 
         if settings.POST_DONATE_NEWSLETTER_URL is not None:
+            # This will send the email address to Mailchimp for Thunderbird only.
+            # TODO: Check if this is even being used.
```

Comment:
Not sure who to ask about this. We should double check with someone to make sure this is still being used for Thunderbird. I suspect it is, but it's good to know for sure.

Reply:
This is definitely still being used, the env variable is set in thunderbird production and staging